apis/refs/v1beta1/helper.go

```diff
@@ -30,3 +29,16 @@ func GetResourceID(u *unstructured.Unstructured) (string, error) {
 	}
 	return resourceID, nil
 }
+
+// TODO(yuhou): Location can be optional. Use provider default location when it's unset.
+func GetLocation(obj *unstructured.Unstructured) (string, error) {
```

Comment:
Done! I think we'll eventually make `GetLocation` a helper function because it will be used in resolving more references. I agree that we need to do so when it's fully functional.